### PR TITLE
Consuming bitso endpoints

### PR DESCRIPTION
**Pull request by timgivois**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js",
+    "dev": "nodemon --ignore db.json --watch index.js"
+  },
+  "author": "Sebastián Osorio <sir@galleto.xyz>",
+  "license": "MIT ",
+  "dependencies": {
+    "async-await": "^0.1.40",
```
Comment (sirgalleto) [inline package.json]: Why are you using `async-await` and `asyncawait` packages?

**Review by sirgalleto — COMMENTED** [review]
Thank you for your PR. 

This approach is really useful, to finalize it you should deprecate the scrapper and the library use to scrap. Also, I leave a comment about the packages.

File: package.json (hunk 1 above)
Comment (timgivois) [reply package.json]: lol, forgot to take them out.

File: index.js [hunk 2]
```diff
@@ -52,7 +54,7 @@ async function start() {
 
         db.setState(prices);
     } catch (e) {
-        console.error('There was a problem updating the values');
+        console.error('There was a problem updating the values', e);
```
Comment (sirgalleto) [inline index.js]: What do you think about using the message attribute instead of all the error object?

**Review by sirgalleto — CHANGES_REQUESTED** [review]
Please help me deprecating the unused code.

File: index.js (hunk 2 above)
Comment (timgivois) [reply index.js]: It's better to use the error name and message.

**Comment by timgivois** [comment]
The scraper is now deprecated

File: services/bitso/api-handler.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const request = require('request-promise');
```
Comment (sirgalleto) [inline services/bitso/api-handler.js]: `request-promise` is not in the `package.json` and I'd like to use the [native promise](https://github.com/request/request-promise-native) version.

File: index.js [hunk 4]
```diff
@@ -2,26 +2,21 @@ const path = require('path');
...
-const scrapCurrencies = require('./currencyScraper');
 const db = require('./db');
+const getBitsoExchange = require('./services/bitso');
 
 const increaseIcon = 'increase.png';
 const decreaseIcon = 'decrease.png';
 
 const timeToUpdate = 60000;
 const currencies = [
-    'btc_mxn',
```
Comment (sirgalleto) [inline index.js:13]: Please put all the currencies in the array again. I'll work on making it more configurable.

File: currencyScraper.js [hunk 5]
```diff
@@ -1,25 +0,0 @@
...
-    if (!data.lastPrices) {
-        throw new Error('No data');
-    }
-
-    const lastPrices = JSON.parse(data.lastPrices);
-
-    console.info(`Price at ${new Date()}:`);
-
-    Object.keys(lastPrices).forEach((currency) => {
-        console.info(
```
Comment (sirgalleto) [inline currencyScraper.js:19]: Can you implement a log report of this one? Helps a lot when you're analyzing the historic.

**Review by sirgalleto — CHANGES_REQUESTED** [review]

File: index.js (hunk 4 above)
Comment (timgivois) [reply index.js:13]: sorry, deleted this...

File: services/bitso/api-handler.js (hunk 3 above)
Comment (timgivois) [reply services/bitso/api-handler.js]: done

**Review by sirgalleto — APPROVED** [review]
Good job!

**Pull request merged**